components/nav/NavTitle.vue

```diff
@@ -11,15 +19,14 @@ router.afterEach(() => {
 </script>
 
 <template>
-  <!-- Use external to force refresh page and jump to top of timeline -->
   <div flex justify-between>
     <NuxtLink
       flex items-end gap-4
       py2 px-5
       text-2xl
       focus-visible:ring="2 current"
-      to="/"
-      external
+      to="/home"
```

Comment:
Probably better to be `/`, as the direction will be different because on whether it's logged. Or assigning the value conditionally.